tests/OSPSuite.Core.Tests/Services/PopulationSimulationPKAnalysesToDataTableConverterSpecs.cs

```diff
@@ -16,8 +16,8 @@ public abstract class concern_for_PopulationSimulationPKAnalysesToDataTableConve
 
       protected override void Context()
       {
-         _dimensionFactory= A.Fake<IDimensionFactory>(); 
-         _pkParameterRepository= A.Fake<IPKParameterRepository>();   
+         _dimensionFactory= new DimensionFactory();
```

Comment:
This test would have been easier if the real objects were actually used

Reply:
YES!